src/algolia-projects.json

```diff
@@ -1,4 +1,16 @@
 [{
+    "name": "Atom.io",
```

Comment:
Go for "Atom npm search" and we will change it later on if we have a better name :).
Ideally we should aim at a better name for the package itself like advised by the community.